keystone_performance/load_test.py

```diff
@@ -304,6 +313,75 @@ def notify_done(self):
         self._done = True
 
 
+
+user_info_array = None
```

Comment:
Let's not start using global variables in this module. Make this a member of TestTracker since that's where it's used.

Otherwise, create a new class for UserInfo and pass an instance into the TestTracker.